pystan/model.py

```diff
@@ -64,10 +66,17 @@ def _map_parallel(function, args, n_jobs):
             import multiprocessing.pool
         except ImportError:
             multiprocessing = None
-        if sys.platform.startswith("win") and PY2:
-            msg = "Multiprocessing is not supported on Windows with Python 2.X. Setting n_jobs=1"
+        if (platform.system() == 'Windows') and PY2:
+            msg = 'Multiprocessing is not supported on Windows with Python 2.X. Setting n_jobs=1'
```

Comment:
I still think this is really unpythonic. We should raise an exception, not modify the user's arguments. Did you forget to change this?

Also, isn't this a Python 3 and a Python 2 issue? Why is there a PY2 check?

Reply:
This is only a python 2 issue. Python 3 users can use `__name__ == '__main__'` trick

Reply:
And only on windows

Reply:
I didn't realize Python 2 users couldn't do the __main__ trick.

Reply:
Sorry, I thought you were introducing the behavior of modifying the user's arguments in this pull request. I gather that was already there. 

But again, please correct me if I'm wrong, it doesn't seem like the changes here have anything to do with the pull request title.

Reply:
Yes, these are not directly related to 2.22, I was just updating the call. Let's fo this in another PR.

Reply:
Yes, these are not directly related to 2.22, I was just updating the call. Let's fo this in another PR.

Reply:
> BTW... 2.23 is about there - likely this week. As it includes reduce_sum you may want to jump straight to that release? Either directly in this PR or as a new PR prior to releasing this.

I think it's good to take it slow. Aren't there other changes we would need to make for 2.23? Is using stanc3 required?

Reply:
This is now reset, I just left ( " --> ' )